.gitlab/ci/global.yml

```diff
@@ -68,3 +68,156 @@
   extends:
     - .default-cache
     - .default-before_script
+
+.notify-slack:
+  after_script:
+    - echo "=====Notify Slack====="
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
+
+run-unittests-and-lint:
+  needs: []
+  stage: unittests-and-validations
+  rules:
+    - if: '$FORCE_BUCKET_UPLOAD != "true" && $INSTANCE_TESTS != "true"'
```

Comment:
```suggestion
```
I think we can get rid of the rule, with the idea that each job which extends from this one will set its own rules.